### ISSUE TITLE
I would like the user to be able to specify the size of the pop-up icons

### ISSUE THREAD

**Issue by MyAlfisti** [FilipePS/Traduzir-paginas-web#511]
The default popup icon size is too small and I would like the user to be able to specify the size of the popup icon.
It would be great if the size of the pop-up icon could be specified in pixels on the Appearance tab of the extension's configuration screen.

**Comment by Slawomir1959** [comment]
i agree with you and i would be also glad to have this possibility to make  this popup icon bigger ..
